cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/LoginCommandOptions.java

```diff
@@ -10,19 +10,19 @@
  */
 public class LoginCommandOptions {
 
-    @Option(names = {"-u", "--user"}, required = false)
+    @Option(names = {"-u", "--user"}, required = false, description = "Your email/username.")
     String userName;
 
-    @Option(names = {"-p", "--password"}, required = false)
+    @Option(names = {"-p", "--password"}, required = false, description = "Your password of your cf account.")
     String password;
 
-    @Option(names = {"-a", "--api"}, required = true)
+    @Option(names = {"-a", "--api"}, required = true, description = "The api of you cf instance.")
```

Comment:
Typing mistake. It  should be called API Endpoint

Reply:
Also it should be `your`, too.

```suggestion
    @Option(names = {"-a", "--api"}, required = true, description = "Your CF instance's API endpoint URL.")
```

@yenle2501 can we say "API endpoint URL", too, or is it not a URL but a name (then I'd use `name` instead of `URL`)?

Reply:
we can also do the same like in this document
[]((https://cli.cloudfoundry.org/en-US/cf/login.html)

Reply:
Changed accordingly.